app/uk/gov/hmrc/plasticpackagingtax/returns/connectors/SubscriptionConnector.scala

```diff
@@ -50,13 +50,6 @@ class SubscriptionConnector @Inject() (
           response
       }
       .andThen { case _ => timer.stop() }
-      .recover {
-        case exception: Exception =>
-          throw DownstreamServiceError(
-            s"Failed to retrieve subscription details for PPTReference: [$pptReferenceNumber], error: [${exception.getMessage}]",
-            exception
-          )
-      }
```

Comment:
We don't want to wrap with `DownstreamServiceError` as this makes it awkward to spot the 404s in the client controller.